pkg/server/server.go

```diff
@@ -238,10 +239,18 @@ func (s *Server) getTarget(target string) *Target {
 
 // Wrapper function to simplify writting error message responses. This function
 // is only intended to be used with the service and nothing else.
-func writeErrorResponse(w http.ResponseWriter, format string, a ...any) error {
+func writeErrorResponse(w http.ResponseWriter, logServer bool, format string, a ...any) error {
 	errmsg := fmt.Sprintf(format, a...)
-	log.Error().Msg(errmsg)
-	http.Error(w, errmsg, http.StatusInternalServerError)
+	if logServer {
+
+		log.Error().Msg(errmsg)
+	}
+	bytes, _ := json.Marshal(map[string]any{
+		"level":   "error",
+		"time":    time.Now().Unix(),
+		"message": errmsg,
+	})
+	http.Error(w, string(bytes), http.StatusInternalServerError)
 	return fmt.Errorf(errmsg)
```

Comment:
Since this function returns an error, we probably want to check it in calls to this function. I would imagine that if an error is occurring here, something really bad has happened.